pyflakes/checker.py

```diff
@@ -1139,15 +1146,15 @@ def IMPORTFROM(self, node):
                 # Only Python 2, local import * is a SyntaxWarning
                 if not PY2 and not isinstance(self.scope, ModuleScope):
                     self.report(messages.ImportStarNotPermitted,
```

Comment:
I prefer just changing the message to show the import statement given that it's apparently common to have different interpretations of `..*`.